packages/cli/src/oclif/commands/promote.js

```diff
@@ -126,8 +146,8 @@ Issues are indicated by ${colors.bold.underline(
     );
 
     const isFeatureUpdate =
-      hasAppChangeType(appMetadata, 'FEATURE-UPDATE') ||
-      hasAppChangeType(issueMetadata, 'FEATURE-UPDATE');
+      hasAppChangeType(appMetadata, 'FEATURE_UPDATE') ||
+      hasAppChangeType(issueMetadata, 'FEATURE_UPDATE');
```

Comment:
We use `FEATURE_UPDATE` everywhere - this looked like it was evaluated to be false every time?

Reply:
How so? Can you explain more?

Reply:
Sure, when we compute the `appMetadata` and `issueMetadata` in `changelog.js`, the `app_change_type` ([line](https://github.com/zapier/zapier-platform/pull/653/files#diff-e9b406434a1740f2bb644fccd54985ae522bb28bd98baef5616478747f5f310cR43)) is one of `BUGFIX` or `FEATURE_UPDATE` ([line](https://github.com/zapier/zapier-platform/pull/653/files#diff-e9b406434a1740f2bb644fccd54985ae522bb28bd98baef5616478747f5f310cR56))

So when it gets evaluated here, in `hasAppChangeType` ([line](https://github.com/zapier/zapier-platform/pull/653/files#diff-c8f46eb46114ddecc4f63cc3ebddd1394fb57c37dfece031f1c2c31bf15a0b6cR40)), the `FEATURE-UPDATE` string we originally had would always evaluate to `false`